docs/editor_manual/finding_your_way_around/account_settings.rst

```diff
@@ -0,0 +1,21 @@
+Account Settings
+~~~~~~~~~~~~~~~~
+
+.. image:: ../../_static/images/screen44_account_settings.png
+
+
+Set the current time zone on a per user basis
+=============================================
+
+Logged-in users can set their current timezone from ``/admin/account/``.
+By default, Wagtail provides the ``pytz.common_timezones`` list of time zones.  This is a list of useful, current timezones. It doesn’t contain deprecated zones or historical zones, except for a few which are deemed as being in common usage, such as US/Eastern.
+
+It is possible to override this list via the :ref:`WAGTAIL_USER_TIME_ZONES <WAGTAIL_USER_TIME_ZONES>` setting.
+
+In case there is zero or one time zone permitted, the form will be hidden.
+
+If there is no time zone selected by the user, the ``LANGUAGE_CODE`` wil be used.
+
+Note that time zones are only applied to datetime fields, not to plain time or date fields.  This is a Django design decision.
+
+
```

Comment:
The "Editor Manual" section of the docs is aimed at individual editors using Wagtail, not developers, so it's not really the right place to refer to Django settings or things applying "on a per-user basis". Since there's not much to be said about the editor-facing behaviour of this feature (beyond "use the time zone dropdown to choose your time zone"...), and we haven't been particularly rigorous about documenting these peripheral bits of Wagtail in the editor manual, I think it's fine to leave this out (and just keep the developer-facing documentation in the Advanced Topics -> Settings section).